qcengine/programs/gamess/harvester.py

```diff
@@ -319,6 +322,95 @@ def harvest_outfile_pass(outtext):
             logger.debug("matched dft xc")
             qcvar["DFT XC ENERGY"] = mobj.group(1)
 
+        # Process EFP
+        mobj = re.search(
+            r"^\s+"
+            + r"ELECTROSTATIC ENERGY  ="
+            + r"\s+"
+            + NUMBER
+            + r"\s*"
+            + r"^\s+"
+            + r"REPULSION ENERGY      ="
+            + r"\s+"
+            + NUMBER
+            + r"\s*"
+            + r"^\s+"
+            + r"POLARIZATION ENERGY   ="
+            + r"\s+"
+            + NUMBER
+            + r"\s*"
+            + r"^\s+"
+            + r"--------------------------------------"
+            + r"\s*"
+            r"^\s+" + r"FINAL EFP ENERGY      =" + r"\s+" + NUMBER + r"\s*",
+            outtext,
+            re.MULTILINE,
+        )
+        if mobj:
+            logger.debug("matched efp")
+            print("matched efp")
+            qcvar["EFP ELECTROSTATIC ENERGY"] = mobj.group(1)
+            qcvar["EFP REPULSION ENERGY"] = mobj.group(2)
+            qcvar["EFP POLARIZATION ENERGY"] = mobj.group(3)
```

Comment:
nice for now, may have to revisit in future. i'm against a 3rd labels set. https://github.com/loriab/pylibefp/blob/master/pylibefp/wrapper.py#L961-L966

Reply:
Noted, that's going to be a headache later on